config/registry/aws/modules/cloudfront-distribution.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "cloudfront-distribution"
+linkTitle: "cloudfront-distribution"
+date: 2021-11-16
+draft: false
+weight: 1
+description: Set up a cloudfront distribution to serve static files from your s3 bucket
+---
+
+This module sets up a cloudfront distribution for you, tailored towards serving static websites/files from an Opta s3 
+bucket (currently just for a single S3, but will expand for more complex usage in the future). Now, hosting your 
```

Comment:
Create a new section here "Linking with s3" then add
```
Once you link the cloudfront module with s3, it can be used to serve static files from your s3 bucket. See below for an example.
```